common/src/autogluon/common/utils/utils.py

```diff
@@ -5,21 +5,40 @@
 import os
 import platform
 import sys
+import numpy as np
```

Comment:
Is this import used?

Reply:
fixed in d9a0a536c5789afe115707770f61df1932ab83ee